pkg/storj/path.go

```diff
@@ -3,5 +3,22 @@
 
 package storj
 
+import (
+	"strings"
+)
+
 // Path represents a object path
 type Path = string
+
+// SplitPath splits path into a slice of path components
+func SplitPath(path Path) []string {
+	if path == "" {
```

Comment:
The special case isn't necessary.

Reply:
I was able to get rid of this in the latest commit.